### PR TITLE
Stop using the combiner in the back end

### PR DESCRIPTION
**Pull request by leofeyer**
Fixes #5626

This PR removes the combiner from the back end and uses Webpack to build the theme CSS files as discussed on the developers meeting.

### CODE REVIEW

**Comment by ausi** [comment]
Should we try to move the three remaining Gulp tasks (minify-theme-css, minify-theme-icons and minify-dark-theme-icons) to Webpack as well?

**Comment by leofeyer** [comment]
Yes, but preferably in a separate PR. There are still some issues with the Encore implementation (see https://github.com/symfony/webpack-encore/issues/349) and we have to figure out how to have source and target images in the same directory.

**Review by ausi — DISMISSED** [review]

File: webpack.config.js [hunk 1]
```diff
@@ -12,6 +12,32 @@ Encore
...
+
+Encore
+    .setOutputPath('core-bundle/contao/themes/flexible')
+    .setPublicPath('/system/themes/flexible')
+    .setManifestKeyPrefix('')
+    .cleanupOutputBeforeBuild(['*.css', '*.json'])
+    .disableSingleRuntimeChunk()
+    .enableSourceMaps()
+    .enableVersioning()
+    .configureCssLoader(config => {
```
Comment (m-vo) [inline webpack.config.js:26]: Do we need to add the (probably transient dependency) `css-loader` to the `package.json`?

**Review by m-vo — DISMISSED** [review]

File: webpack.config.js (hunk 1 above)
Comment (leofeyer) [reply webpack.config.js:26]: Added in 7e826e1124001e2768883da5409c8421fa4e454f.

**Review by m-vo — APPROVED** [review]

**Review by ausi — APPROVED** [review]

**Comment by aschempp** [comment]
> There are still some issues with the Encore implementation (see [symfony/webpack-encore#349](https://github.com/symfony/webpack-encore/issues/349))

what's the issue with this PR? Why do we need SVGO?

**Pull request merged**